build/builder.go

```diff
@@ -568,13 +568,25 @@ func (b *Builder) Finish() error {
 
 		delete(toDelete, final)
 
-		b.shardLog("upsert", final)
+		b.shardLog("upsert", final, b.opts.RepositoryDescription.Name)
 	}
 	b.finishedShards = map[string]string{}
 
 	for p := range toDelete {
+		if zoekt.TombstonesEnabled(filepath.Dir(p)) {
+			// Don't delete compound shards, set tombstones instead.
+			if strings.HasPrefix(filepath.Base(p), "compound-") {
+				if strings.HasSuffix(p, ".zoekt") {
+					repoName := b.opts.RepositoryDescription.Name
+					b.shardLog("set_tombstone", p, repoName)
```

Comment:
minor: more minimal name, like `tombstone` or even better `tomb` so that it is under 8 chars for tabs to align nicely :O?

Reply:
"tomb" it is